Simplenote/src/main/java/com/automattic/simplenote/StyleActivity.java

```diff
@@ -47,14 +47,8 @@
     private LinearLayoutManager mLayoutManager;
```

Comment:
We can remove the `import androidx.core.app.NavUtils;` statement since it is no longer used.